lua/harpoon/cmd-ui.lua

```diff
@@ -0,0 +1,148 @@
+local harpoon = require("harpoon")
+local popup = require("popup")
+local log = require("harpoon.dev").log
+local term = require("harpoon.term")
+
+local M = {}
+
+Harpoon_cmd_win_id = nil
```

Comment:
I know I wrote this in the past, but I hate these global vars.  I feel like we shouldn't be using them :)

But we are currently, so I'll let it slide, but if I could get my way it would be to take these pop-up windows and just turn it into a class.